files/ko/glossary/bootstrap/index.html

```diff
@@ -0,0 +1,22 @@
+---
+title: 부트스트랩
+slug: Glossary/Bootstrap
+tags:
+  - 부트스트랩
+  - bootstrap
+  - Glossary
+  - framework
+  - css
+translation_of: Glossary/Bootstrap
+---
+<p>Bootstrap은 반응형 웹사이트를 빠르게 만들기 위한 무료 오픈소스 {{Glossary("HTML")}}, CSS 및 {{ Glossary ( " JavaScript " ) }} 의 프레임워크 입니다.</p>
```

Comment:
```suggestion
<p>Bootstrap은 반응형 웹사이트를 빠르게 만들기 위한 무료 오픈소스 {{Glossary("HTML")}}, CSS 및 {{Glossary( "JavaScript")}} 의 프레임워크 입니다.</p>
```
매크로가 작동하지 않는 오류가 있습니다.